Druid-Tests/DruidTestInterpreter.class.st

```diff
@@ -718,7 +718,7 @@ DruidTestInterpreter >> primitiveFloat64At [
 	addressToRead := self stackTop.
 	self
 		pop: 1
-		thenPush: (objectMemory float64At: addressToRead) rawFloatBits
```

Comment:
Mea culpa, I introduced that in the druid experimental branch but never fixed it in slang.

I prefer that floats understand `rawFloatBits` instead of using  `rawFloatBitsOf:...`. What do you think?
Obtaining the bits of a float is something that is independent of the representation of objects, so ideally it should not be in the objectMemory